services/catalog/tests/unit/test_services_compatibility.py

```diff
@@ -279,5 +279,5 @@ async def test_evaluate_service_compatibility_map_with_deprecated_versions(
         compatibility_map[ServiceVersion("1.0.0")] is None
     )  # cannot upgrade to deprecated 1.0.1
     assert compatibility_map[ServiceVersion("1.0.1")] is None  # Deprecated version
-    assert compatibility_map[ServiceVersion("1.2.0")].can_update_to == "1.2.5"
+    assert compatibility_map[ServiceVersion("1.2.0")].can_update_to.version == "1.2.5"
```

Comment:
Q: where do you test updating to a different key?

Reply:
I second the question

Reply:
@sanderegg @GitHK  see `async def test_evaluate_service_compatibility_map_with_other_service` in the same file